docs/changelog/changelog_en.md

```diff
@@ -1,5 +1,81 @@
 # Changelog plentyshopPWA
 
+## v1.4.0 (2024-04-08) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0" target="_blank" rel="noopener">
+
+### Migration guide
+
+- The upload action was changed [.github/workflows/upload.yml](https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0#diff-8cf0b5fae548aab9bd49f5020da9c0e35d281984b82b8a5358ffb1c1ae5bec13L5) update the file to make use of the [config feature](https://pwa-docs.plentymarkets.com/guide/setup/deployment#config)
+- We now require an access token to make requests to the plentymarkets api. [guide](todo)
+- Category routing has been updated, and the /c prefix has been removed. Please verify that no static URLs in your application still include /c.
+  - To accommodate the /c routing change, the [category page](https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0#diff-2f61484eb978aa090fc50dcba90bc44813b45081f25dbff295434cdf6bf219a4) was moved from apps/web/pages/category/[slug].vue to apps/web/pages/[...slug].vue.
+
+
+### New
+
```

Comment:
We added the following:

Reply:
Is this supposed to replace "New"? Then it would break with the Format of all other changelog entries made to the PWA yet. @plentyofDennis

Reply:
We leave it as is for now